model_loader: # must much the node name in launch file
  ros__parameters:
    model_name: "array_net_model"
    model_path: "/home/kunwu/Documents/ros_ws/src/network_loader/models/"
    input_dims: 5
    output_dims: 5

